rerun/src/main/java/org/apache/falcon/rerun/handler/AbstractRerunConsumer.java

```diff
@@ -88,7 +88,6 @@ public void run() {
                     LOG.warn("Entity {} of type {} doesn't exist in config store. Rerun "
                                     + "cannot be done for workflow ", message.getEntityName(),
                             message.getEntityType(), message.getWfId());
-                    return;
```

Comment:
It should be continue else the message below would get logged. (Error in rerun consumer) ?